### ISSUE TITLE
Should there be labels for testing?

### ISSUE THREAD

**Issue by m3rc3n4ry** [jackyko1991/vnet-tensorflow#2]
From what I know, there should be no labels along testing data otherwise it's just training data. So, as explained in the folder hierarchy, why are labels provided with their corresponding images in the training folder? (I have not read the full paper yet)

**Comment by jackyko1991** [comment]
The testing data is a subset of training data to evaluate the training model.
